### PR TITLE
Feature/claims navigation/add new damage picker screen

### PR DESCRIPTION
**Pull request by juliaendre**
## [APP-XXX]

- Location picker screen
- Damage picker screen
- Brand/model picker screen
- Created generic checkbox picker

Figma: https://www.figma.com/file/QdEYhDOuTTci4AUCldDvqt/Navigation?type=design&node-id=695-11837&t=7S9g4e4uSRxARVA5-0

## Checklist

- [x] Dark/light mode verification
- [x] Landscape verification
- [x] iPad verification
- [x] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/hCoreUI/Sources/hForm/hRow.swift [hunk 1]
```diff
@@ -69,6 +70,12 @@ public struct hRow<Content: View, Accessory: View>: View {
         return new
     }
 
+    public func withoutDivider() -> Self {
```
Comment (sladan-hedvig) [inline Projects/hCoreUI/Sources/hForm/hRow.swift:73]: Ditch this, better use enivorment property

File: Projects/hCoreUI/Sources/hForm/hRow.swift [hunk 2]
```diff
@@ -46,6 +46,7 @@ public struct hRow<Content: View, Accessory: View>: View {
     var accessory: Accessory
     var horizontalPadding: CGFloat = 21
     var verticalPadding: CGFloat = 21
+    var showDivider: Bool = true
```
Comment (sladan-hedvig) [inline Projects/hCoreUI/Sources/hForm/hRow.swift:49]: Better switch this to environment property and create modifier to set it to disable but default value should be true.
Later on, you can put that modifier on hSection level and all hRow inside section with inherit that modifier.

File: Projects/hCoreUI/Sources/hForm/hRow.swift [hunk 3]
```diff
@@ -105,6 +112,15 @@ public struct StandaloneChevronAccessory: View {
     }
 }
 
+public struct StandaloneChevronNewAccessory: View {
```
Comment (sladan-hedvig) [inline Projects/hCoreUI/Sources/hForm/hRow.swift:115]: Merge this with StandaloneChevronAccessory
and add environment hUseNewDesign property to decide if you want to use it.

File: Projects/hCoreUI/Sources/hForm/hRow.swift [hunk 4]
```diff
@@ -105,6 +112,15 @@ public struct StandaloneChevronAccessory: View {
...
+    public init() {}
+
+    public var body: some View {
+        Image(uiImage: hCoreUIAssets.chevronRight2.image)
+            .foregroundColor(hLabelColorNew.tertiary)
```
Comment (sladan-hedvig) [inline Projects/hCoreUI/Sources/hForm/hRow.swift:120]: Remove foregroundColor. This should be on decided on where it was called upon

File: Projects/hCoreUI/Sources/hForm/hRow.swift [hunk 5]
```diff
@@ -139,6 +164,12 @@ extension hRow {
         }
     }
 
+    public var withNewChevronAccessory: hRow<Content, ChevronNewAccessory> {
```
Comment (sladan-hedvig) [inline Projects/hCoreUI/Sources/hForm/hRow.swift:167]: I think you can ditch this after you merge ChevronNewAccessory and ChevronAccessory

File: Projects/hCoreUI/Sources/hForm/hRow.swift [hunk 6]
```diff
@@ -139,6 +164,12 @@ extension hRow {
...
+        hRow<Content, ChevronNewAccessory>(ChevronNewAccessory()) {
```
Comment (sladan-hedvig) [inline Projects/hCoreUI/Sources/hForm/hRow.swift:168]: Also this

**Review by sladan-hedvig — CHANGES_REQUESTED** [review]

File: Projects/Claims/Sources/Views/SubmitClaim/SingleItem/CheckboxPickerScreen.swift [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+import hCore
+import hCoreUI
+
+struct CheckboxPickerScreen<T>: View {
+    var items: [(object: T, displayName: String)]
+    let preSelectedItems: () -> [T]?
+    let onSelected: ([T]) -> Void
+    let onCancel: () -> Void
+    let oneValueLimit: Bool?
+    let smallerPadding: Bool?
```
[the remark this answers is not in the document]
Comment (julia-andersson) [reply Projects/Claims/Sources/Views/SubmitClaim/SingleItem/CheckboxPickerScreen.swift:11]: @sladan-hedvig but that is not the same thing?

**Review by sladan-hedvig — APPROVED** [review]

**Pull request merged**